consensus/consensus.go

```diff
@@ -149,3 +152,10 @@ type Istanbul interface {
 	// SetChain sets chain of the Istanbul backend
 	SetChain(chain ChainReader)
 }
+
+type ConsensusInfo struct {
```

Comment:
Do we need to redefine this structure?

Reply:
ah, it is moved from `consensus/istanbul/backend/api.go`? Is that right?

Reply:
Yes, right.